Gaming The Syllabus/Assets/DebugTextManager.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class DebugTextManager : Singleton<DebugTextManager>
+{
+    [SerializeField] private TextMeshProUGUI textElement;
+
+    private void Awake()
+    {
+        InitializeSingleton();
+    }
+
+    public static void Log(string text)
+    {
+        _instance.textElement.text = text;
+    }
+}
```

Comment:
I'm assuming this is for making debugging easier and cleaner. Looks good!

Reply:
Not just easier and cleaner but possible really :( 
Android doesn't have good debugging support so this seems like the best I could do...